suite-native/transactions/src/screens/TransactionDetailScreen.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import { Box, Divider } from '@suite-native/atoms';
+import {
+    AccountsStackParamList,
+    AccountsStackRoutes,
+    Screen,
+    ScreenHeader,
+    StackProps,
+} from '@suite-native/navigation';
+import { selectBlockchainState, selectTransactionByTxid } from '@suite-common/wallet-core';
+import { formatNetworkAmount, getConfirmations } from '@suite-common/wallet-utils';
+
+import { TransactionDetailHeader } from '../components/TransactionDetail/TransactionDetailHeader';
+import { TransactionDetailData } from '../components/TransactionDetail/TransactionDetailData';
+import { TransactionDetailConfirmations } from '../components/TransactionDetail/TransactionDetailConfirmations';
+import { TransactionDetailAmount } from '../components/TransactionDetail/TransactionDetailAmount';
+
+export const TransactionDetailScreen = ({
+    route,
+}: StackProps<AccountsStackParamList, AccountsStackRoutes.TransactionDetail>) => {
+    const { txid } = route.params;
+    const transaction = useSelector(selectTransactionByTxid(txid));
+    const blockchain = useSelector(selectBlockchainState);
+
+    if (!transaction) return null;
```

Comment:
Can you please add `// TODO` common that we should add some kind of empty state UI here? Also mention this in that task for empty states we already have in github.

Reply:
added